app/src/main/java/org/zalando/nakadi/config/NakadiResourceServerTokenServices.java

```diff
@@ -42,9 +42,11 @@ public OAuth2Authentication loadAuthentication(final String accessToken) throws
             if (!featureToggleService.isFeatureEnabled(Feature.REMOTE_TOKENINFO)) {
                 try {
                     return localService.loadAuthentication(accessToken);
-                } catch (final OAuth2Exception ex) {
+                }
```

Comment:
lets keep formatting the same across the file (in the other part it uses different)